src/pages/QuestionPage.tsx

```diff
@@ -35,24 +60,28 @@ export default function QuestionPage() {
       justifyContent="center"
     >
       <Flex alignItems="center" maxW="md" margin="4">
-        {prompt && answers && index ?
+        {gotQuestion && prompt && answers && index ?
           <VStack>
             <Logo size={["32px", "50px"]} />
             <Question
               id="1"
               title={`Question ${index ? `#${index}` : ""}`}
               text={prompt ? prompt : ""}
             />
-            <Answer setAnswer={handleSetAnswer} id="1" background="red" text={answers?.at(0)} />
-            <Answer setAnswer={handleSetAnswer} id="2" background="blue" text={answers?.at(1)} />
-            <Answer setAnswer={handleSetAnswer} id="3" background="green" text={answers?.at(2)} />
-            <Answer setAnswer={handleSetAnswer} id="4" background="orange" text={answers?.at(3)} />
-            <Button onClick={submitAnswer} alignSelf="end">Submit</Button>
+            <Answer setAnswer={handleSetAnswer} id="0" background="red" text={answers?.at(0)} />
+            <Answer setAnswer={handleSetAnswer} id="1" background="blue" text={answers?.at(1)} />
+            <Answer setAnswer={handleSetAnswer} id="2" background="green" text={answers?.at(2)} />
+            <Answer setAnswer={handleSetAnswer} id="3" background="orange" text={answers?.at(3)} />
+            {isHost ?
+              <Button onClick={nextQuestion} alignSelf="end">Next Question</Button>
```

Comment:
This button needs to be more clear/more apparent on the page. Consider adding some styling to this button to make it stand out

Reply:
added some drippy styling for you ;)